tests/unit/fixtures/gameStates/cardValidationErrors.js

```diff
@@ -0,0 +1,196 @@
+import { Card } from '../Card';
+import GamePhase from '../../../../utils/GamePhase.json';
+import MoveType from '../../../../utils/MoveType.json';
+
+
+// jack of clubs appears twice
```

Comment:
I have been looking on my phone and had difficulty finding the duplicate card. The comment says jack of clubs, but the test seems to look for seven of spades? Can you comment the places where the card is duplicated so it's easy to find?